app/extend/context.js

```diff
@@ -65,12 +65,14 @@ module.exports = {
    * @method Context#getFileStream
    * @param {Object} options
    *  - {Boolean} options.requireFile - required file submit, default is true
+   *  - {Object} options.multipart - override default multipart configurations
    * @return {ReadStream} stream
    * @since 1.0.0
    */
   async getFileStream(options) {
     options = options || {};
-    const parts = this.multipart({ autoFields: true });
+    options.multipart = options.multipart || {};
+    const parts = this.multipart(Object.assign({ autoFields: true, options.multipart }));
```

Comment:
> options are passed to busboy. The only additional option is autoFields.

`getFileStream()` only work on `autoFields = true`, so you can't change it.

Reply:
So we can do the following

```js
    const parts = this.multipart(Object.assign({ }, options.multipart ,{autoFields: true}));

```

Reply:
@fengmk2 what about
```js
    const parts = this.multipart(Object.assign({ }, options.multipart ,{autoFields: true}));
 ```

Reply:
@fengmk2 I opend https://github.com/eggjs/egg-multipart/pull/25